INFRASTRUCTURE.md

```diff
@@ -0,0 +1,77 @@
+# Our Infrastructure
+
+First, let's address some details of this document.
+
+<details>
+  <summary>What, why, who...</summary>
+
+  ## What?
+
+  This document outlines the various systems that JSON Schema the organisation uses and maintains.
+  This does not include social media profiles.
+
+  ## Why?
+
+  It's part of the required tasks to be completed by the OpenJS Foundation, forming an item of the [onboarding checklist](https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboaghrding-checklist).
+  It's also probably good practice.
+
+  This enables members of the JSON Schema team, and the OpenJS Foundation, to have oversight and management of such infrastructure should it be required.
+
+  ## Who?
+
+  This was mainly a list created by Ben Hutton (@relequestual).
+  Others are free to amend and update this (via Pull Requests) as is required.
+
+</details>
+
+JSON Schema mostly operates on GitHub and Slack.
+
+Our GitHub organisation is located at https://github.com/orgs/json-schema-org.
+
+Our Slack server is located at https://json-schema.slack.com.
+(Anyone can join using https://json-schema.org/slack.)
+
+The JSON Schema organisation maintains ONE domain name: json-schema.org.
+
+The domain name is registered and paid for by the OpenJS Foundation.
+
+The domain name defers its name servers to CloudFlare.
+The DNS mostly points to GitHub.
+
+The website is split into several applications for historical reasons.
+
+The main website is run through GitHub Pages. The source code is located at https://github.com/json-schema-org/json-schema-org.github.io.
+
+The human friendly documentation for the specification, dubbed "Understanding JSON Schema", is located at https://json-schema.org/understanding-json-schema.
```

Comment:
Idly -- we have full control over DNS right? Maybe we should get slightly cute and route https://understanding.json-schema.org to the book?

Reply:
Yes. We could totally do that. Although, we're part way through merging all the sites together under one static build site.